### ISSUE TITLE
Add resources section to learn more about ConstraintLayout

### ISSUE THREAD

**Issue by amardeshbd** [amardeshbd/android-constraint-layout-cheatsheet#102]
Not sure if this would be benefitial, however, curious mind may jump into one of the video resources if they see it.

Type of resources:
* Video
  * Youtube link
  * Title
  * Summary
* Blog
  * Title
  * Author info
  * URL

**Comment by amardeshbd** [comment]
So far added follow resources to firebase firestore.

```

author
summary
title
event
type 
= "blog", "techtalk"
url
publish_date


------------------------------


ConstraintLayout Deep Dive
(Android Dev Summit '18)


Android Developers
Published on Nov 7, 2018

https://www.youtube.com/watch?v=P9Zstbk0lPw

In this session you will learn best practices of using ConstraintLayout on Android, particularly covering tips and tricks in the Layout Editor and new features introduced in the 2.0 version. We will go over how to take advantage of those to create UI more efficiently.

Presented by: Sean McQuillan & Chris Sinco


Android Dev Summit '18 all sessions playlist → http://bit.ly/ADS18-Sessions
Subscribe to the Android Channel! → http://bit.ly/AndroidDevs1

#AndroidDevSummit

------------------------


URL: 
https://www.youtube.com/watch?v=ytZteMo4ETk

#io18
What's new with ConstraintLayout and Android Studio design tools (Google I/O '18)


Android Developers
Published on May 10, 2018

This session will cover the new features in ConstraintLayout 2.0, as well as the latest functionality added in Android Studio design tools, highlighting how to effectively take advantage of them for designing, prototyping, and building a graphical user interface application.

Rate this session by signing-in on the I/O website here → https://goo.gl/a7Krho

Watch more Android sessions from I/O '18 here → https://goo.gl/R9L42F
See all the sessions from Google I/O '18 here → https://goo.gl/q1Tr8x

Subscribe to the Android Developers channel → http://goo.gl/GEh1ds


-------------------

https://www.youtube.com/watch?v=29gLA90m6Gk

What's New in ConstraintLayout (Google I/O'19)
Android Developers
Published on May 9, 2019

Learn the capabilities of ConstraintLayout, a popular library for UI development on Android, which addresses common pitfalls of and solutions to UI patterns, performance aspects, integrating motion, and how to best take advantage of the library in your own application through a series of use cases and examples.        

Watch more #io19 here: 
Android & Play at Google I/O 2019 Playlist → https://goo.gle/2GRWlQh
Google I/O 2019 All Sessions Playlist → https://goo.gle/io19allsessions 
Learn more on the I/O Website → https://google.com/io

Subscribe to the Android Developers Channel → https://goo.gle/AndroidDevs
Get started at → https://developer.android.com/

Speaker(s): Nicolas Roard and John Hoford

T7F6C5

------------------------

URL: https://www.youtube.com/watch?v=rzmB3UxxhaA

Mastering ConstraintLayout in Android



GDG Johannesburg
Published on Oct 5, 2017
In this talk, Rebecca looks at the different types of layouts that are available for you in Android. Specifically focusing on ConstraintLayout, this talk will guide you through using most of the features of ConstraintLayout in a practical demonstration. Components such as Barriers, Chains, Guidelines and Constraints are covered in this talk as well as some performance tools you can use to create great layouts in Android.
Category
Science & Technology


------------------------
URL: 
https://www.youtube.com/watch?v=EBke5RoKK0M

ConstraintLayout 2.0 by Nicolas Roard and John Hoford, Google EN

Android Makers, France




Android Makers
Published on May 5, 2019
This session was given at Paris Android Makers 2019 by Nicolas Roard and John Hoford, Google.

More info: http://androidmakers.fr/schedule/?ses...

ConstraintLayout is a popular library for UI development on Android. It is deeply integrated with the Android Studio design tools, allowing developers to quickly create complex and reactive UIs. We recently released ConstraintLayout 2.0, which brings even more layout capabilities, virtual viewgroups, performance improvements and flexible ways to organize your UI. More uniquely, the library also provides powerful animation and motion handling capabilities via MotionLayout, a subclass of ConstraintLayout. All together, this new declarative approach makes it very easy to create modern, motion-enabled UIs for your application.

This talk will cover the new capabilities of the library, common pitfalls and solutions to UI patterns, performance aspects, integrating motion, and how to best take advantage of the library in your own application via a series of use cases and examples.

Alex Karp Production


------------------------
URL: 



------------------------
URL: 
```

**Comment by amardeshbd** [comment]
![device-2019-06-06-190330](https://user-images.githubusercontent.com/99822/59073743-6dcaf400-8896-11e9-9a97-84d3ee384dee.png)
